.travis.yml

```diff
@@ -0,0 +1,28 @@
+# we want to make sure it is run on trusty where sudo is provided for postgres
+dist: trusty
+language: python
+python:
+    - "3.6"
+sudo: required
+cache:
+  directories:
+    - eggs
+install:
+  - pip install -e .[test]
+  - pip install flake8 codecov
+  - sleep 5
+before_script:
+  - docker pull spotify/kafka
+  - docker run -p 9999:9092 --name kafka -d spotify/kafka
+  - sleep 10
+  - docker ps -a
+  - docker exec kafka /opt/kafka_2.11-0.10.1.0/bin/kafka-topics.sh --create --zookeeper localhost:2181 --topic T-UserAction --replication-factor 1 --partitions 1
```

Comment:
the `--topic` value does not have to be T-UserAction but it does not matter what the topic name is as long as the test set up creates a topic that the producer and consumer can work with

Reply:
@ableeb if you look at the tests, I'm producing to and consuming from arbitrary topics, and that seems to just work.

Any idea why?